src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -204,3 +206,115 @@ def validate_infrastructure_type(infra):
         if s.lower() == infra.lower():
             return s
     return "generic"
+
+
+def get_values_file():
+    values_file_provided = False
+    values_file = os.getenv('HELMVALUESPATH')
+    if (values_file is not None) and (os.path.isfile(values_file)):
```

Comment:
what does this return if file does not exist?

Reply:
it returns a flag and file, if file not exists it return False, None